api.py

```diff
@@ -14,6 +14,10 @@
 CORS(app)
 citation_graph = CitationNetwork()
 
+DEFAULT_SERIALIZATION_ARGS = {
+    "exclude": [Cluster.searchable_case_name, ],
+}
+
```

Comment:
Should DEFAULT_SERIALIZATION_ARGS be set as default parameters in model_list_to_json, and then optionally manually overriden? This seems like a cleaner way to do this instead of repeating **DEFAULT.... everywhere.

Reply:
Yeah that makes sense. The only problem is that L44 uses `model_to_dict`, the non-list form of that helper (which is provided by peewee directly) and I wasn't sure what the best way to share the serialization configuration was.

I miss my jbuilders :(

Reply:
Actually, I can pass that manually to `model_to_dict` but make it default for the list helper.

Reply:
Fixed.